src/types/modules/react-native.d.ts

```diff
@@ -283,7 +283,10 @@ declare module 'react-native' {
         enterKeyHint?: 'enter' | 'done' | 'go' | 'next' | 'previous' | 'search' | 'send';
         readOnly?: boolean;
     }
-    interface TextInputProps extends WebTextInputProps {}
+    interface TextInputProps extends WebTextInputProps {
+        smartInsertDelete?: boolean;
```

Comment:
Could you add a `TODO` that we can remove `smartInsertDelete` once we upgrade the App to RN 0.73? Thanks!